### PR TITLE
Slovak (UTF-8) translation

### PR DESCRIPTION
**Pull request by michalholes**
I translated Marlin to Slovak language.

### CODE REVIEW

File: Marlin/language_sk_utf8.h [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+ * See also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
+ *
+ * Translated by Michal Holeš, Farma MaM
+ * http://www.facebook.com/farmamam
+ *
+ */
+#ifndef LANGUAGE_SK_UTF_H
+#define LANGUAGE_SK_UTF_H
+
+#define MAPPER_C3C4C5_SK
```
Comment (thinkyhead) [inline Marlin/language_sk_utf8.h:37]: This mapper needs to be added to `utf_mapper.h` for full support (currently falls back to `MAPPER_NON`).

File: Marlin/language_sk_utf8.h [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+#define DISPLAY_CHARSET_ISO10646_SK
```
Comment (thinkyhead) [inline Marlin/language_sk_utf8.h:38]: This CHARSET should be added to the list in `language.h`, at least as a placeholder if there won't be any custom font needed. However, it appears like this language would benefit from a custom font (for characters like `ť`).

**Review by thinkyhead — COMMENTED** [review]
Some characters seem unlikely to be found in the built-in font. Does this need a mapper and font?

**Comment by thinkyhead** [comment]
I've put together #7556 which adds placeholders for the bits that could be added in future.

**Comment by michalholes** [comment]
Works fine. Thanks.

**Pull request closed without merging**